server/actions/user.js

```diff
@@ -228,13 +244,15 @@ exports.userList = {
       offset: offset
     }
 
-    if (!data.session.user.isAdmin) {
+    if (!data.session.user.isAdmin && data.session.user.role !== 'moderator') {
```

Comment:
`data.session.user.role !== 'moderator'` seems better if changed to `data.session.user.isModerator`